peps/pep-0772.rst

```diff
@@ -0,0 +1,1618 @@
+PEP: 772
```

Comment:
Please use 773, https://github.com/python/peps/pull/4218 pipped you to 772 by a couple of hours. 
```suggestion
PEP: 773
```